.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: "✨ feature"
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
```suggestion
**Is your feature request related to a problem? Please describe it.**
```

Reply:
That's the default template suggestion from GitHub 😉